app/controllers/declaration/DeclarationHolderController.scala

```diff
@@ -66,16 +64,21 @@ class DeclarationHolderController @Inject()(
 
   private def addAnotherYesNoForm: Form[YesNoAnswer] = YesNoAnswer.form(errorKey = "declaration.declarationHolders.add.another.empty")
 
-}
-
-object DeclarationHolderController {
+  private def continueToPageOnJourney(implicit request: JourneyRequest[_]): Mode => Call =
```

Comment:
This method's name is not explanatory enough. I'd suggest renaming it so that it is clear that it's about redirecting when there are no holders in cache.

Reply:
Done